### PR TITLE
Add changes from reviews

### PR DESCRIPTION
**Pull request by dilanSachi**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: examples/http-client-header-parameter/http_client_header_parameter_header.bal [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    artist: "John Coltrane"
+};
+
+map<string> headers = {
+    "Accept": mime:APPLICATION_JSON
+};
+
+// Sends a `POST` request to the "/albumId" resource.
+// The query parameter can be provided as parameters in the `post` method invocation.
+string albumId = check albumClient->/addAlbum.post(payload, headers);
```
Comment (shafreenAnfar) [inline examples/http-client-header-parameter/http_client_header_parameter_header.bal:12]: In this case also lets give the headers in line

File: examples/http-client-query-parameter/http_client_query_parameter_post.bal [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Album payload = {
+    title: "Blue Train",
+    artist: "John Coltrane",
+    year: 1958
+};
+
+// Sends a `POST` request to the "/addAlbum" resource.
+// The query parameter can be provided as parameters in the `post` method invocation.
+string albumId = check albumClient->/addAlbum(payload, rating = 4.8);
```
Comment (shafreenAnfar) [inline examples/http-client-query-parameter/http_client_query_parameter_post.bal:9]: string albumId = check albumClient->/album(payload, rating = 4.8);

Comment (dilanSachi) [reply examples/http-client-query-parameter/http_client_query_parameter_post.bal]: Updated

File: examples/http-client-header-parameter/http_client_header_parameter_header.bal (hunk 1 above)
Comment (dilanSachi) [reply examples/http-client-header-parameter/http_client_header_parameter_header.bal]: Updated

**Review by shafreenAnfar — APPROVED** [review]

**Pull request merged**